src/convertToObject.js

```diff
@@ -3,16 +3,22 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const listOfStyles = sourceString
+    .split(';')
+    .map(el => el.split(':'))
+    .filter(el => el.length > 1);
+
+  return listOfStyles.reduce((styles, [key, value]) => {
+    return {
+      ...styles,
+      [key.trim()]: value.trim(),
+    };
+  }, {});
```

Comment:
```suggestion
  return listOfStyles.reduce((styles, [key, value]) => ({
    ...styles,
    [key.trim()]: value.trim(),
  }), {});
```

or you can check here key and value existing and delete the filter from getting listOfStyles (-1 cycle)